src/trio/_core/_run.py

```diff
@@ -554,6 +564,12 @@ def __enter__(self) -> Self:
                 "Each CancelScope may only be used for a single 'with' block",
             )
         self._has_been_entered = True
+        assert self._is_relative is not None
+
+        if self._is_relative:
+            self._deadline = current_time() + self._relative_deadline
+            self._is_relative = None
```

Comment:
need to dedent this

Reply:
not relevant since removal of `self._is_relative`